jib-core/src/main/java/com/google/cloud/tools/jib/api/TarImage.java

```diff
@@ -17,80 +17,73 @@
 package com.google.cloud.tools.jib.api;
 
 import java.nio.file.Path;
+import java.util.Optional;
+import javax.annotation.Nullable;
 
 /**
  * Builds to a tarball archive.
```

Comment:
"Builds" doesn't apply anymore. (Or, does it still from the end-user perspective?)

Reply:
I think I'll make these comment changes in the other PR since this one doesn't cover base image support.